packages/ui-time-select/src/TimeSelect/README.md

```diff
@@ -15,11 +16,13 @@ render: true
 <TimeSelect
   renderLabel="Choose a time"
   onChange={(e, { value }) => console.log(value)}
+  onHideOptions={(e)=> console.log("hide opts")}
```

Comment:
Is this needed?

Reply:
I think its useful because it shows users the order of the events firing